pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_config_repo.adoc

```diff
@@ -0,0 +1,29 @@
+[id="config_repo_{context}"]
```

Comment:
s " "config_repo_{context}" "/" "configuring-a-repository-as-a-content-source-for-pantheon_{context}"